index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Portfolio: Ashley Whitted</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lobster|Sacramento|Satisfy|Tangerine" rel="stylesheet">
+</head>
+
+<!-- linked stylesheet to style.css 
+create page header, then, using nav, created home, project, resume links to start body -->
+
+<body>
+   <header class="pageHeader">
+     <h1>Ashley Whitted</h1>
+     <nav>
+       <ul class="flexNav">
+         <li class="flexNavList"><a href="index.html">HOME</a></li>
+         <li class="flexNavList"><a href="projects.html">PROJECTS</a></li>
+         <li class="flexNavList"><a href="resume.html">RESUME</a></li>
+       </ul>
+    </nav>
+    </header>
+
+    <img Id="headshot" src="./images/headshot.jpg" alt="myPhoto">
+
+    <div class="flexIntro">
+        <h2 class="flexIntroItem">Hi! I'm Ashley....</h2>
+        <br>
```

Comment:
It's fine to use `<br>` every once in a while, but your HTML should not be responsible for the layout or positioning of elements on the page beyond placing them in their normal flow as inline or block elements. You don't need it here where you are using block elements (`<p></p>`).